### ISSUE TITLE
Spotting ShotSpotters

### ISSUE THREAD

**Issue by Weihua4455** [themarshallproject/arpa20220125#91]
Mainly for fun!

**Comment by eads** [comment]
In Syracuse.

**Comment by ghing** [comment]
- Map of ShotSpotters (and one ALPR) spotted in Syracuse: [ShotSpotter Spotting Syracuse](https://gist.github.com/ghing/03f02224b8487af1810136db7531859f)
- Photos of the devices: https://photos.app.goo.gl/LJGniV8NUjNa2hBr9

**Comment by red-bin** [comment]
(those aren't actually shotspotter devices AFAIK. A lot of the equipment there is ~400MHz radio gear, probably for their closed network, with "public safety" equipment. It's something we saw in Chicago where a lot of OEMC camera equipment was paired with similar looking radio equipment that they use for internal communication, but not actually shotspotter. There's a chance it is, since shotspotter's branched out into realtime analytics outside of "just" gunshot detection... but just be careful that it's accurate. At LPL we rented a telescopic lens to take photos of the devices' labeling to pull FCC/contract/bar code details to confirm.)

**Comment by red-bin** [comment]
Yeah, shotspotter makes it intentionally difficult to know what devices look like, so it's pretty challenging. 

We were only able to identify one model. I've heard that they have more than one model, but I haven't seen any direct confirmation of that fact, like what we were able to get with the telescopic lens. 

Have a look at https://observablehq.com/d/0dbc293ec5abfecd -- the second map (the one with fewer dots) has all the shotspotter devices we found. The thing to look for are the two-stack devices, with the top device being the sensor and the bottom being the radio device.

Interestingly, the FCC only has one Shotspotter device that's received certification. There's a chance that a subcontractor or external company filed FCC certs for future devices.
